ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml

```diff
@@ -96,5 +96,9 @@ ref:
     default: ""
     documentation: |-
       Allow users to make S3 deletion optional
+  - name: USER_PROVISIONED_DNS
+    default: ""
```

Comment:
If we go with a "yes"/"no" pair, I think it's better for the default to be set to `"no"`